apps/qubit/modules/object/actions/addDigitalObjectAction.class.php

```diff
@@ -69,6 +69,25 @@ public function execute($request)
       $this->forward404();
     }
 
+    // Assemble resource description
+    sfContext::getInstance()->getConfiguration()->loadHelpers(array('Qubit'));
+
+    if ($this->resource instanceof QubitActor)
+    {
+       $this->resourceDescription = render_title($this->resource);
+    }
+    elseif ($this->resource instanceof QubitInformationObject)
+    {
+      $this->resourceDescription = '';
+
+      if (isset($this->resource->identifier))
+      {
+        $this->resourceDescription .= $this->resource->identifier .' - ';
```

Comment:
I think we adopted Symfony's style for the `.` separator (no spaces around it) but I personally prefer spaces across all operators. Either way, one or the other ;)

Reply:
Cool... I'll do the spaces around them.